authoring-new-actions.md

```diff
@@ -0,0 +1,90 @@
+
+Note: All actions published from **Azure** GitHub org are branded using the Azure icon irrespective of the icon used in the action.yml. This is to leverage the recall value of Azure brand and icon as compared to the individual icons of various Azure services. 
+
+## Action Versioning
+
+**Versioning Guidance for GH actions that should be adhered to before a new action or an enhancement to an existing action is released:** 
+
+Official Release Management Guidance from GH platform team: https://docs.github.com/en/free-pro-team@latest/actions/creating-actions/about-actions#using-release-management-for-actions
+
+**Note:** Introduce a new major version of the branch (releases/v2) and tag (v2) if changes break the existing workflow. Ensure to move the major version tag to always point to the most recent minor version tag to ensure all the existing users of the major version would automatically gain the benefits from the new enhancements added in the minor version release.
+
+
+## Publish the action to Marketplace
+
+### Pre-requisites:
+
+- Ensure that the Legal OSS approval process for the repo is complete. Search for the status of the work item created with your repo name here: https://ossmsft.visualstudio.com/Reviews/\_workitems/ .
+
+&quot;DO NOT PUBLISH THE ACTION TO THE MARKETPLACE UNTIL LEGAL REVIEW IS COMPLETE&quot;
+
+- Versioning guidelines mentioned above section are adhered to and you have a version tag
+- Have a readme.md in place with documentation
+- Include V-team for review process and sign off by mailing@ [V team Alias](mailto:)
+
+Finally [publish the action to GitHub Marketplace](https://docs.github.com/free-pro-team@latest/actions/creating-actions/publishing-actions-in-github-marketplace)
+  - Typically for all actions intended to &quot;deploy to Azure&quot;, we mark Primary Category as &quot;Deployment&quot;
+  - Secondary Category can be chosen based on the intent of the Action
+- Give a suitable release title and write up. Tip: Include a sample workflow and License details
+- &quot;Publish release&quot;
+- Navigate to [https://github.com/marketplace?type=actions](https://github.com/marketplace?type=actions) and search for your Action to ensure everything is right in place.
+
+### Include a starter template in the [Azure/actions-workflow-sample](https://github.com/Azure/actions-workflow-samples)s repo
+
+Various starter templates are made available to help users easily get started with GitHub Actions for Azure and deploy their apps created with popular languages and frameworks such as .NET, Node.js, Java, PHP, Ruby, or Python in containers or running on any operating system.
+
+It is recommended that a suitable workflow template is added to this repo: [https://github.com/Azure/actions-workflow-samples](https://github.com/Azure/actions-workflow-samples) by raising a PR into the relevant folder.
+
+For more help, please reach out to @ [V Team Alias](mailto:)
```

Comment:
Here too, please add the team name and alias

Reply:
done